java-client/src/test/java/org/scassandra/http/client/BatchExecutionTest.java

```diff
@@ -0,0 +1,13 @@
+package org.scassandra.http.client;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.junit.Test;
+
+public class BatchExecutionTest {
+    @Test
+    public void testEquals() throws Exception {
+        EqualsVerifier.forClass(BatchExecution.class)
```

Comment:
Yeah that and https://github.com/MutabilityDetector/MutabilityDetector are two if the first things on my class path for a Java project